Getting-Started/Setup/Upgrading/upgrading-to-v8.md

```diff
@@ -0,0 +1,80 @@
+
+### Database types supported
+
+As Umbraco 8 doesn't support MySQL databases the migration will not work when moving from an Umbraco 7 site using MySQL to Umbraco 8 on SQL server for example.
+The database types that are supported are SQL Server and SQL CE.
+
+### Third party property editors
+
+The migration will transform the data stored in third party editors as well, however it will simply be stored as it was in Umbraco 7. If the structure has changed for Umbraco 8 or the property editor doesn't exist you will still be able to find the data in the database, but you would not see it in the backoffice.
+
+## What will happen?
+
+When the migrations are running, Umbraco will go through your entire Umbraco 7 database and update it to the format required for Umbraco 8. The schema will be remodelled and transformed into the correct format and your existing compatible data will also be transformed to fit with how Umbraco 8 is storing data.
+
+Please be aware that these migrations will be running directly on your database. They are transforming schema and data - not transferring. Therefore always ensure that you have a backup before attempting to do this. In case something goes wrong, you will be able to rollback and try again.
+
+## How does it work?
+
+In the following guide we will try to migrate the content of an Umbraco 7.13.1 site to Umbraco 8.1.0.
+
+### Step 1: Upgrading to 7.14
+
+Before the content migration can start the site has to be 7.14+. A good idea would be to **always take a database backup** before doing an upgrade, and then check the [version specific upgrade instructions](version-specific.md).
+
+The site in this example is an Umbraco 7.13.1 site, and we will use Nuget to update it.
+
+It is a fairly simple site, with a bit of content:
+
+![v7 site with content](images/v7-content.png)
+
+Following the [general upgrade instructions](general.md) we will now upgrade via Nuget until we get to this point:
+
+![Upgrading to v7.14](images/upgrading-7.14.png)
+
+Once it is upgraded and you have verified everything is working, move on to step 2.
+
+### Step 2: Migrating content to Umbraco 8
+
+First thing to do is spin up a fresh new Umbraco 8.1+ site. Make sure it all works and that no content is there (_hint:_ choose not to install starter kit).
+
+![Fresh 8.1 site](images/fresh-8.1-site.png)
+
+Now you should take a backup of your database from the **Umbraco 7.14 site**. You can then add the information for the backup database and add that to the connection string for the  **Umbraco 8.1 site**. Do note that if you are running SQL CE, you will have to copy the database over to the new site as well.
+
+Once the connection string is set, the final step is to change the Umbraco version number in the web.config on the **Umbraco 8.1 site**, to say 7.14.0. This will mean that it thinks there is an upgrade as the Umbraco dlls are 8.1 but the config says 7.14 it needs to run the upgrade.
+
```

Comment:
I think we should consistently call it a migration when it's going from V7 to V8 and not an upgrade - obviously ok to say 7.13->7.14 is an upgrade.

So would change this to say

it needs to run the migration.